src/com/studentmanagementsystem/presentation/Main.java

```diff
@@ -0,0 +1,85 @@
+package com.studentmanagementsystem.presentation;
+
+import com.studentmanagementsystem.data.list.*;
+import com.studentmanagementsystem.logic.*;
+import com.studentmanagementsystem.presentation.*;
+
+public class Main {
+
+    public static void main(String[] args)
+        {
+            final int ADD_STUDENT_MENU_ITEM = 1;
+            final int UPDATE_STUDENT_MENU_ITEM = 2;
+            final int REMOVE_STUDENT_MENU_ITEM = 3;
+            final int SHOW_STUDENTS_MENU_ITEMS = 4;
+            final int EXIT_MENU_ITEM = 5;
+            final int NO_MENU_ITEM_SELECTED = 0;
+
+
+            Menu menuItems = new Menu();
+            StudentDAO studentDao = new ArrayListStudentDAO();
+            UserInputReader inputReader = new UserInputReader();
+            int chosenMenuItem = NO_MENU_ITEM_SELECTED;
+
+            while (chosenMenuItem != EXIT_MENU_ITEM) {
+                menuItems.show();
+                chosenMenuItem = inputReader.chooseMenuItem();
+
+                switch (chosenMenuItem) {
+                    case ADD_STUDENT_MENU_ITEM:
+
+                        String studentFirstName = inputReader.readStudentFirstName();
+                        String studentLastName = inputReader.readStudentLastName();
+                        int studentId = inputReader.readStudentId();
+
+                        Student newStudent = new Student();
+                        newStudent.setStudentFirstName(studentFirstName);
+                        newStudent.setStudentLastName(studentLastName);
+                        newStudent.setStudentId(studentId);
+                        studentDao.add(newStudent);
+                        break;
+
+                    case UPDATE_STUDENT_MENU_ITEM:
+                        for (Student currentStudent : studentDao.getAll()){
```

Comment:
This for loop you use twice (see case REMOVE_STUDENT_MENU_ITEM) -> maybe move this to a method and only call the method?